node/as/thrift.js

```diff
@@ -23,6 +23,7 @@
 var assert = require('assert');
 var bufrw = require('bufrw');
 var Result = require('bufrw/result');
+var thriftify = require('thriftify');
```

Comment:
I think this means you need to shift a dev dep -> prod dep? or is that already done?